src/components/shared/DetailsWidget/types.ts

```diff
@@ -7,7 +7,7 @@ import {
 /*
  * Which icons correspond to which action types in the details widget
  */
-export const ACTION_TYPES_ICONS_MAP: Record<AnyActionType, string> = {
+export const ACTION_TYPES_ICONS_MAP: Partial<Record<AnyActionType, string>> = {
```

Comment:
The partial here was so that we would not need to repeat ourselves, I have since removed it and decided to go down the route of repeating ourselves because it is the simplest way to get the safe values to work with the current setup. Also doing so gives us more flexibility for future designs which might differentiate between different safe interactions with different logos